### ISSUE TITLE
Focus outline shows on non-focusable elements on IE 11

### ISSUE THREAD

**Issue by ncksllvn** [uswds/uswds#2928]
## Description
Using IE11, the focus outline works as expected when using the tab key. However, if I click a non-focusable element on the page, the dotted outline shows even though it's not focusable.

## Steps to reproduce the issue

1. Open IE11
2. Navigate to site
3. Click a non-focusable element, such as a div container or image
4. See outline appear

## Additional information [optional]

- This issue was discovered during an accessibility audit on VA.gov.
- Screenshot after clicking "Getting started" container
![image](https://user-images.githubusercontent.com/1915775/53964004-6a1fa980-40bc-11e9-89e3-2e7f62dcb953.png)

**Comment by maya** [comment]
Hi, thanks for opening this issue. We'll review this with our team and see how it fits on our roadmap. Stay tuned!

**Comment by ncksllvn** [comment]
Thank you for checking it out @maya!
